### ISSUE TITLE
start() rejects

### ISSUE THREAD

**Issue by kristiandupont** [testcontainers/testcontainers-node#215]
I am trying to start a container like this:
```js
    const genericContainer = new GenericContainer(
      'kristiandupont/dvdrental-image'
    )
      .withExposedPorts(5432)
      .withEnv('POSTGRES_PASSWORD', 'postgres')
      .withStartupTimeout(timeout);
    const startedContainer = await genericContainer.start();

    const stream = await startedContainer.logs();
    stream
      .on('data', (line) => console.log(containerLogPrefix + line))
      .on('err', (line) => console.error(containerLogPrefix + line))
      .on('end', () => console.log(containerLogPrefix + 'Stream closed'));
```

Now, the call to `start()` fails with this message:
```
 Log stream ended and message "Started!" was not received for c3b5e5ba7d71d93e40c8a3fdefce0d0bbc1d2d432ef0d1b3230e7ee79f023185
```

As there is no stream established yet, I don't know how I can receive any information to debug this problem further?

**Comment by cristianrgreco** [comment]
Hi @kristiandupont, as per the Configuration section of the README you can see all output by setting the `DEBUG=testcontainers*` environment variable. For example `DEBUG=testcontainers* npm test`

**Comment by kristiandupont** [comment]
My apologies, I missed that. I got it working though -- it turns out that even though it appears that the Ryuk image has been updated to multiarch, my maching (arm64 linux) tried to run the x86 version. I am not sure why, but when disabling ryuk it works. 

A possible improvement could be to log something if ryuk fails to start because I spent a long time under the assumption that the problem was with my own image.
